### PR TITLE
JADE: Options menu

### PR DESCRIPTION
**Pull request by Nostritius**
This PR adds the basic options menu for Jade Empire.

### CODE REVIEW

File: src/engines/jade/gui/main/options.cpp [hunk 1]
```diff
@@ -173,6 +175,11 @@ void OptionsMenu::callbackActive(Widget &widget) {
 		return;
 	}
 
+	if (widget.getTag() == "CREDITS") {
+		playVideo("creditmovie");
+		return;
+	}
+
```
[the remark this answers is not in the document]
Comment (Nostritius) [reply src/engines/jade/gui/main/options.cpp]: No, Jade has actually two way of showing the credit video, by a button in the main menu and by a button this button in the options menu.

[the remark this answers is not in the document]
Comment (DrMcCoy) [reply src/engines/jade/gui/main/options.cpp]: Ah, okay :)

**Comment by Nostritius** [comment]
All issues should now be fixed.

File: src/engines/jade/gui/main/options.cpp [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+	_backButton = getButton("ButtonBack");
+	_currentButton = 0;
+
+	_optionsDescription = getLabel("OptionsDescLabel");
+
+	_audioOptionsDescription = TalkMan.getString(129);
+	_videoOptionsDescription = TalkMan.getString(130);
+	_difficultyOptionsDescription = TalkMan.getString(152);
+	_gameInfoOptionsDescription = "";
```
Comment (DrMcCoy) [inline src/engines/jade/gui/main/options.cpp]: In English, that menu item is called "Feedback". The TLK string is 153.

Comment (Nostritius) [reply src/engines/jade/gui/main/options.cpp]: This are only the descriptions, the text which appears in the box below the options when hovering ofer the buttons. It was for the Feedback/gameinfo menu empty when hovering over it (at least in the german version).

Comment (DrMcCoy) [reply src/engines/jade/gui/main/options.cpp]: Well, that's a bug in the German version then. The English version does have a text there and it's 153.

Comment (DrMcCoy) [reply src/engines/jade/gui/main/options.cpp]: untitled

**Comment by Nostritius** [comment]
I've now changed the empty string to the 153 string and recognized, that this string is empty (at least in the german version

**Comment by DrMcCoy** [comment]
Merged as 33635c48...af924c29, thanks! :)

**Pull request closed without merging**